### PR TITLE
[REF] *: remove read_group usage

### PR DESCRIPTION
**Pull request by ryv-odoo**
read_group will be deprecated soon, replace read_group usage from the the business code with _read_group.

https://github.com/odoo/enterprise/pull/72217

### CODE REVIEW

File: addons/website_membership/controllers/main.py [hunk 1]
```diff
@@ -62,25 +62,27 @@ def members(self, membership_id=None, country_name=None, country_id=0, page=1, *
...
+            if not any(country.id == country_id for country, __ in country_groups):
+                country_groups = [(g_country, count) for g_country, count in country_groups if g_country]
+                country_groups.append((Country.browse(country_id).sudo(), 0))
+                country_groups = sorted(country_groups, key=lambda c, __: c.name or '')
+
+        countries = [{
+            'country_id_count': sum(count for __, count in country_groups),
+            'country_id': (0, _("All Countries")),
+        }]
+        for g_country, count in country_groups:
```
Comment (kmagusiak) [inline addons/website_membership/controllers/main.py:81]: Nitpicking: you can make a single comprehension for `countries = [{}, *({} for ...)]`
Same in other website modules.

**Review by kmagusiak — APPROVED** [review]
LGTM, just fix what the runbot says (deprecated method, etc.)

File: addons/website_membership/controllers/main.py (hunk 1 above)
Comment (ryv-odoo) [reply addons/website_membership/controllers/main.py:81]: I tried but IMO, it less readable (really just an opinion).

**Comment by ryv-odoo** [comment]
@robodoo r+

**Pull request closed without merging**